Dockerfile

```
@@ -1,71 +1,106 @@
+CMD ["pnpm", "exec", "prisma", "migrate", "deploy"]
+
+# Standalone Next.js runtime.
+FROM ${NODE_IMAGE} AS app
+ARG NEXT_BUILD_ID
+WORKDIR /app
+ENV NODE_ENV=production \
+    HOSTNAME=0.0.0.0 \
+    PORT=3000 \
+    APP_VERSION=${NEXT_BUILD_ID} \
+    NEXT_TELEMETRY_DISABLED=1
+RUN apt-get update \
```

Hadolint (2.15.1)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)